src/components/Gauge.react.js

```diff
@@ -48,24 +51,33 @@ class Gauge extends React.Component {
 
     const progress = computeProgress({ min, max, value, progressionTarget: 1 });
 
+    const elementName = getClassName('gauge', theme);
+
     const currentValue = (
       <CurrentValue
+        className={elementName + '__current-value'}
         valueColor={colorValue}
         units={units}
         css={'transform: translateY(-150%); top: 0;'}
       >
         {logarithmic ? log.formatValue(value, base) : value.toFixed(1)}
       </CurrentValue>
     );
+    const filteredProps = getFilteredProps(this.props);
 
     return (
-      <div id={id} className={className} style={style}>
+      <div id={id} className={elementName + ' ' + (className || '')} style={style}>
```

Comment:
```suggestion
      <div id={id} className={elementName + (className ? (' ' + className) : '')} style={style}>
```
(and similar in a few other places)